pkg/resources/backup/aws_test.go

```diff
@@ -60,7 +60,7 @@ func TestAWSSnapshotPostgres(t *testing.T) {
 	}()
 
 	err = executor.PerformBackup(client, time.Second*10)
-	if err != nil {
```

Comment:
can you explain this further?

Reply:
Thanks for comment @MStokluska . I planned to return to this issue. Tried to investigate now, and opened the task/Bug, for more investigation and fix - https://issues.redhat.com/browse/MGDAPI-6388

Reply:
I commented my changes (to ignore timeout issue), so now TestAWSSnapshot* tests are failing

Reply:
The issue fixed. Fake Client with WithStatusSubresource used for Snapshots* (4 places) - https://github.com/valerymo/integreatly-operator/blob/MGDAPI-6289/pkg/resources/backup/aws_test.go#L34
Thanks for help with `WithStatusSubresource`